renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      "automerge": true,
+      "matchUpdateTypes": [
+        "patch",
+        "pin",
+        "digest"
+      ]
+    },
+    {
+      "groupName": "terraform",
+      "managers": [
+        "terraform"
+      ],
+      "matchUpdateTypes": [
+        "minor"
+      ]
+    }
+  ],
+  "prConcurrentLimit": 2,
+  "schedule": [
+    "every weekday",
```

Comment:
Current dependabot pattern is to update weekly on Monday mornings. For testing on this repo is fine, but will want to probably loosen it to weekly before rolling it out further.

Reply:
OK. We are going to leave it as-is for now so we get a shorter feedback loop with testing.